tests/python/frontend/keras/test_forward.py

```diff
@@ -54,6 +54,26 @@ def to_channels_last(arr):
             tvm.testing.assert_allclose(kout, tout, rtol=1e-5, atol=1e-5)
 
 
+def test_forward_elemwise_add():
```

Comment:
This is already covered in test_forward_merge https://github.com/dmlc/tvm/blob/master/tests/python/frontend/keras/test_forward.py#L57

It can be removed.